cli/config/compose/profiles/fleet/configurations/kibana.config.yml

```diff
@@ -15,5 +15,6 @@ xpack.fleet.enabled: true
 xpack.fleet.registryUrl: http://package-registry:8080
 xpack.fleet.agents.enabled: true
 xpack.fleet.agents.elasticsearch.host: http://elasticsearch:9200
+xpack.fleet.agents.fleetServerEnabled: true
 xpack.fleet.agents.kibana.host: http://kibana:5601
```

Comment:
You might need `xpack.fleet.agents.fleet_server.hosts: [http://?:5601]` instead here with the most recent builds. Probably worth rerunning.